### PR TITLE
Fix #536

### LINKED ISSUE #536: Add a form to add a new note
### Expected Behaviour

A simple form should be available to add new notes.
The author of the note should be automatically set to the logged-in user.
A _Add new note_ button should be displayed on the note list view.

### Actual Behaviour

It is only possible to create notes through the Django backend.

### PR DESCRIPTION
**Pull request by lingxiaoyang**
### Changes proposed in this pull request:

* Note add view (form, view, template, unittest)
* Client note add view (reused form, view, template, new unittest)

### Status

- [X] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### How to verify this change

* http://localhost:8000/note/  should have "new note" button.
* http://localhost:8000/note/add/  should work and should redirect back to /note/.
* http://localhost:8000/member/view/612/notes  should have "Add a note" button.
* http://localhost:8000/member/view/612/notes/add  should work, should show the name of client, and should redirect back to client note page.

### Deployment notes and migration

No migration is needed for this change

### New translatable strings

- [x] If applicable, I have included updated .po files with new strings (see http://goo.gl/kfBO7N)

### Additional notes

Previews: https://github.com/savoirfairelinux/sous-chef/issues/536

### CODE REVIEW

File: src/note/forms.py [hunk 1]
```diff
@@ -7,3 +7,13 @@ class NoteForm(forms.ModelForm):
     class Meta:
         model = Note
         fields = ['note', 'client', 'priority']
+
+        widgets = {
+            'note': forms.Textarea(attrs={'rows': 5}),
+            'client': forms.Select(
+                attrs={'class': 'ui status_to dropdown'}
```
Comment (manumilou) [inline src/note/forms.py]: Semantic UI provides a searchable dropdown widget, which is more efficient when list has a lot of elements: http://semantic-ui.com/modules/dropdown.html#search-selection.

Also, what is the use of the class _status_to_ ?

File: src/note/forms.py [hunk 2]
```diff
@@ -7,3 +7,13 @@ class NoteForm(forms.ModelForm):
...
+            ),
+            'priority': forms.Select(
+                attrs={'class': 'ui status_to dropdown'}
```
Comment (manumilou) [inline src/note/forms.py]: Same question here: what is the use of the class _status_to_ ?

File: src/member/templates/client/view/notes.html [hunk 3]
```diff
@@ -46,6 +46,7 @@ <h2 class="ui header">
         <div class="field">
             <a href="{{ request.path }}?display={{ display }}" class="ui button">{% trans "Reset" %}</a>
             <button class="ui yellow button" type="submit">{% trans "Search" %}</button>
+            <a href="{% url 'member:client_notes_add' client.pk %}" class="ui primary">{% trans "Add a Note" %}</a>
```
Comment (manumilou) [inline src/member/templates/client/view/notes.html]: We should use the same type of link that the note list. Something like:
```
<a href="{% url 'note:note_add' %}" class="ui  button">
  <i class="add icon"></i>{% trans 'New note' %}
</a>
```

Comment (lingxiaoyang) [inline src/member/templates/client/view/notes.html]: Will be corrected in next commit:
image

Comment (lingxiaoyang) [inline src/member/templates/client/view/notes.html]: Fixed in 479b430:
image

File: src/note/forms.py (hunk 1 above)
Comment (lingxiaoyang) [inline src/note/forms.py]: Good idea.
I made an error. I removed `status_to` in 479b430.

**Pull request merged**